README.md

```diff
@@ -1,2 +1,86 @@
-# nf-synstage
-Nextflow pipeline for staging files from Synapse
+# nf-synstage: Stage Synapse Files
+
+## Purpose
+
+The purpose of this Nextflow workflow is to automate the process of staging Synapse files in an accessible location (_e.g._ an S3 bucket). In turn, these staged files can be used as input for a general-purpose (_e.g._ nf-core) workflow that doesn't contain Synapse-specific steps for downloading data. This workflow is intended to be run first in preparation for other data processing workflows.
+
+Briefly, `nf-synstage` achieves this automation as follows:
+
+1. Extract all Synapse URIs (_e.g._ `syn://syn28521174`) from a given text file
+2. Download the corresponding files from Synapse in parallel
+3. Replace the Synapse URIs in the text file with their staged locations
+4. Output the updated text file so it can serve as input for another workflow
+
```

Comment:
Should there be a section on dependencies?  Like installation of tower cli?

Reply:
This README is intended to document the workflow itself. Technically, it can be deployed using the Tower web UI, the Tower CLI, or the Nextflow CLI. For Sage staff, we will document how to deploy workflows on Tower [here](https://sagebionetworks.jira.com/wiki/spaces/WF/pages/2191556616/Getting+Started+with+Nextflow+and+Tower). I will be started with the web UI, but I will likely elaborate on how to use the Tower CLI and document any dependencies there.

Reply:
Got it - i like it. It's my opinion that it could be beneficial to have a short sentence that links to those docs so that users can go to those docs to learn how to deploy this workflow.

Reply:
I'm still torn because listing these deployment options is another thing that we would need to maintain across every Nextflow repository we create. I think identifying this as a Nextflow is sufficient because I can't imagine people landing on this README without knowing how to launch Nextflow workflows.

Reply:
I agree that lots of interlinking of docs can result in a lot of debt/stale links. It might be best to keep the docs more granular for now, and then add these links as the documentation stabilizes/the landscape of what needs to be documented becomes clearer. Maybe this issue could be solved in the interim with a single link to a central ToC that lives on confluence or a github readme, that points to all of these various docs....

Reply:
I like the idea of a documentation hub. It would likely be on GitHub because I don't want to assume that folks reading this README have access to our internal Confluence docs. 

Created a ticket: https://sagebionetworks.jira.com/browse/WORKFLOWS-215